Bottom: fe7d00daba11bb3825db5f5d37fed90deb67e001
Top:    ad83c5fd4cbc295370a90bc09afae45a82f5a783
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 19:21:55 +0000

thermal.c: Add #warning pointing out where we might get a speedup








---

diff --git a/src/thermal.c b/src/thermal.c
index 445b97d..fa87f28 100644
--- a/src/thermal.c
+++ b/src/thermal.c
@@ -432,6 +432,7 @@ ThermPoly (PCBTypePtr p, PinTypePtr pin, Cardinal laynum)
         arc = CirclePoly (pin->X, pin->Y, pin->Thickness / 2);
         /* create a thin ring */
         poly_Boolean_free (pa, arc, &m, PBO_SUB);
+#warning THIS WOULD BE FASTER IF WE JUST SUTRACTED A CROSS SHAPE
         /* fix me needs error checking */
         if (style == 2)
           {
